Zend/tests/property_hooks/parent_get_plain.phpt

```diff
@@ -8,7 +8,7 @@ class P {
 }
 
 class C extends P {
-    public $prop {
+    public $prop = 42 {
```

Comment:
Why do you need these tests changes?
Does the patch break some previous behavior?

Reply:
Yes, this patch also fixes a bug. Properties do not inherit default values, but we accidentally did for properties with hooks.

Reply:
If the fix may be done separately, it's better to do this separately (add bug report and so on).
If the fix relays on new inheritance logic, and can't be simple separated, don't do this.